rocksdb-sys/build.rs

```diff
@@ -13,6 +13,15 @@ fn main() {
 
     let snappy = env::var_os("DEP_SNAPPY_INCLUDE").expect("DEP_SNAPPY_INCLUDE is set in snappy.");
 
+    // NOTE: the cfg! macro doesn't work when cross-compiling, it would return values for the host
+    let target_os = env::var("CARGO_CFG_TARGET_OS").unwrap_or_default();
```

Comment:
Is `CARGO_CFG_TARGET_OS` set automatically when cargo runs, i.e. is it a "standard" env var? What is the default?

Reply:
Yes, it is set by cargo depending on the OS the current build is targeting. This would be the same as `if cfg!(target_os = "android") {}` but the `cfg!` macro doesn't do the right thing when cross-compiling.